Create 08_Tool_Sets.txt

Turtlelord26 opened the pull request:
#932 

Proposed templating for tool set items with examples, in a new, dedicated file. Having tool sets exist makes it easier to add them to profession backgrounds. Also prose is appearing.

Medkit and repair kit should probably move to this page from 07_items, along with a couple other things.

Turtlelord26 committed:
Create 08_Tool_Sets.txt

#932, vaguely.

Proposed templating for tool set items with examples. Having tool sets exist makes it easier to add them to profession backgrounds.

Medkit and repair kit would probably move to this page from 07_items, along with a couple other things.

Turtlelord26 committed:
Add empty weight field

Turtlelord26 committed:
Typography

Turtlelord26 committed:
Alphabetized

Turtlelord26 committed:
Added non-exhaustive list of skeletons

inspired by list of professional backgrounds.

rosebrallier reviewed (commented):
This is a fantastic start!
I'm thinking that when characters attempt a skill check without the necessary tools, they roll at disadvantage, and in some cases may have to roll jury-rigging at disadvantage instead of the more pertinent skill.

rosebrallier commented:
I'm going to have @Reveraine pile on this and fill out some of the prose. @dfuchs2 you can add to this as well, as your time allows. Follow the completed sets as a template.

Reveraine committed:
Added Flavor text to some tool kits

Added text to:
Sculptor's Tools
Musical Instrument and Care Supplies
Digital Artist's Program and Tablet Package

Turtlelord26 committed:
Add Tool Set rules to skill rule block

@bleehu can we move all the skill rules into the skills document?

Skill gain per level was in both, so I killed it here.
Added some tool set rules.

@rosebrallier I don't remember what word we decided to use to describe tool-required skills so it's placeholdered in the initial commit.

Turtlelord26 committed:
More content!

Couple new sets, added skills and noise to sets that didn't have them.

Turtlelord26 committed:
Adding Assisted skill appellations

Some skills may have to be separated out (i.e. Sleight of Hand and Lockpicking) in order to more precisely append the Assisted status.

Turtlelord26 committed:
Edits to tool set rules section

Turtlelord26 commented:
This may be approaching basic functionality @rosebrallier 

It would be helpful to the current iteration of the Assisted Skills mechanic (requires tools) to separate out some skills, such as lockpicking from sleight of hand, so that I can define what can be used without tools from the skills side, but I'm not sure if we want to increase skill granularity like that. I am also hesitant to try and define that from the toolkit end of the mechanic, as it would be harder for players to look up. Maybe it's less of a problem than I think and I should just leave it entirely to the GMs whether players can do [A Thing] with a particular skill and [lack of] tool.

Turtlelord26 committed:
Some text

Bit of prose up in the Assisted description, and by weapon skill.

Turtlelord26 committed:
Context is King

Added context stats to all sets.
Added prose to the research AI.
Added protective cases and battery life to terminals and tablets.

Turtlelord26 committed:
Added descriptor clause for the Context stat

Turtlelord26 committed:
Added default tool bonus

Turtlelord26 committed:
Added 1st iteration prices.

Keep in mind we're gong to be several of these away via backgrounds (and some classes like Hacker who need their tool set to function or Soldiers who would flunk out of boot camp at warp speed for losing their firearm care kit).

Turtlelord26 committed:
Added prose to the trauma kit

Turtlelord26 commented:
Explorers prep would love to have fewer branches to trip people with.

rosebrallier commented on Items/08_Tool_Sets.txt:
```diff
@@ -0,0 +1,208 @@
+and a miniaturized 3D circuit printer allow the professional electrician 
+to maintain, repair, and construct all manner of electronic gadgetry.
+
+Relevant Skill(s): Repair, Jury-rig
```
*Jury-rigging

rosebrallier reviewed (commented):
We've made some darn good progress here. I've even acquired some inspiration for additional prose.
In other news, let's defer adding weight info to any of these yet. If nothing has a weight, it'll be harder to miss one when we develop the relevant systems.

rosebrallier committed:
Blade_maintenance_prose

rosebrallier committed:
Disguise_kit_prose

rosebrallier committed:
Fine_art_kit_prose

rosebrallier commented:
The remaining required prose is not particularly within my areas of interest, experience or expertise.
@bleehu we need your input on a few of these, and well as your review of the prices re: game economy.
@Reveraine See what else you can chuck into the prose pot?
@Turtlelord26 I approve of what you've done with Assisted skills. Gives a framework for the GM and players to play off of without getting too...nitpicky.

rosebrallier commented on CharacterCreation/06_Skills.txt:
```diff
@@ -65,6 +65,11 @@ this is not an exhaustive list of skills, merely the skills called for most
+	Assisted skills, which usually require an appropriate tool set as 
+described in the basic rules, are marked with an [A]. Keep in mind that, regardless 
+of [A]'s, having an appropriate set of tools for the task at hand will help with 
+almost any skill, though.
```
We may want to make an explicit exception here for Reload checks. They're Weapons skill checks, but shouldn't require an entire skill check to perform.

Turtlelord26 committed:
Typo

Turtlelord26 committed:
De-Assisted Weapons skill

Between reload checks and the use of weapons - X to identify weapons in enemy's hands across the battlefield, I think my goals will be better attained through careful management of weapon maintenance DCs.

Turtlelord26 commented on Items/08_Tool_Sets.txt:
```diff
@@ -116,10 +116,12 @@ Noise Level: Low
 
 Relevant Skill(s): Performance
-Context: 2D physical Art (eg. painting)
+Context: Physical fine art (eg. painting, sculpting)
```
See Sculptor's Kit, below

rosebrallier replied:
Herp derp. Not sure why I didn't see that when I was reading the musical instrument entry.

Turtlelord26 commented:
In-scope: Refill kits for the toolsets flavored as containing consumables?

rosebrallier commented:
> In-scope: Refill kits for the toolsets flavored as containing consumables?

Sure; we could duplicate the ammo pricing philosophy.

rosebrallier committed:
2d art

bleehu commented on 01_01_Basic_Rules.txt:
```diff
@@ -563,15 +563,32 @@ have invested in learning the skill; however, your GM may decide otherwise, appl
-
 	Defenders win ties for opposing rolls.
 
+-- Assisted Skills and Tool Sets--
```
Callling a skill "assisted" brings to mind two characters helping on the same skill check, as opposed to a character using a tool. Maybe something like "instrumented skill check"? Though I admit that doesn't have a nice ring to it.

bleehu commented on 01_01_Basic_Rules.txt:
```diff
@@ -563,15 +563,32 @@ have invested in learning the skill; however, your GM may decide otherwise, appl
+
+	Other skills suffer no penalties to use without a tool set, but an 
+appropriate one to the task at hand may still provide a bonus. Unless otherwise 
+specified, augmenting a roll with an appropriate tool set provides +10 to the roll.
```
Good default. I like this.

bleehu commented on 01_01_Basic_Rules.txt:
```diff
@@ -625,4 +642,4 @@ out a receiver, you may carry (at max) one additional receiver in order to swap
 instantly jams the weapon on first shot. Failure on both checks requires an 
-additional check.
\ No newline at end of file
+additional check.
```
EOF changed from windows/linux standard. Not sure why. Shouldn't break anything. Don't really care.

bleehu commented on Items/08_Tool_Sets.txt:
```diff
@@ -0,0 +1,215 @@
+Relevant Skill(s): Weapons - Melee, Weapons - Thrown
+Context: Bladed Weapons
+Noise Level: Low
+Weight: 
```
Let's remove unknown weights and add them back in when we write encumbrance rules?

bleehu commented on Items/08_Tool_Sets.txt:
```diff
@@ -0,0 +1,215 @@
+Noise Level: Low
+Weight: 
+
+Digital Artist's Program and Tablet Package ($250):
```
This is brilliant! Forge documents? Oh ya!

bleehu replied:
Let's drop the price of this just a bit; I expect it to come up fairly infrequently.

bleehu commented on Items/08_Tool_Sets.txt:
```diff
@@ -0,0 +1,215 @@
+Noise Level: Low
+Weight:
+
+Destructive Entry Kit ($200):
```
Should this replace the similar tools in the tool section of items?

Turtlelord26 replied:
My endgame is to absorb that entire section into this document

bleehu commented on Items/08_Tool_Sets.txt:
```diff
@@ -0,0 +1,215 @@
+Weight:
+
+Electronic Lock Spoofing Kit ($250):
+	[ph]
```
Blocking from merging until place holders are filled.

bleehu commented on Items/08_Tool_Sets.txt:
```diff
@@ -0,0 +1,215 @@
+academic or library databanks. Takes time to achieve best results. Comes with 
+a protective case and rechargeable 8-hour battery.
+
+Relevant Skill(s): Knowledge - All, Investigation
```
I'm really torn on this item. On the one hand, it seems OP. On the other hand, it's got a price tag to match and wikipedia is a real thing.

Turtlelord26 replied:
It's also entirely controlled by the GM and basically useless in combat.

bleehu commented on Items/08_Tool_Sets.txt:
```diff
@@ -0,0 +1,215 @@
+Weight: 
+
+Hostile Environment General Purpose Survival Package ($600):
+	[ph][one kit does for a party]
```
Too general; let's have different kits for each hostile environment.

bleehu commented on Items/08_Tool_Sets.txt:
```diff
@@ -0,0 +1,215 @@
+Noise Level: Low
+Weight: 
+
+Field-expedient Trauma Kit ($400):
```
Request change: Let's fold this item and the Doctor's field applicator together into one item. I don't want the Doc to feel like they need to draw two separate tools to get their best healing done.

bleehu replied:
I suggest that medicine and repair stay separate. Blending them here is the first instance of the anti-pattern of combining them.

bleehu reviewed (changes requested):
Pretty cool new system! Gonna need at least one more iteration before it's ready for dev.

bleehu committed:
added prose for firearm tool.

Turtlelord26 committed:
Typography

Turtlelord26 committed:
Assisted -> Intricate

Turtlelord26 committed:
Assisted -> Intricate 2

Turtlelord26 committed:
Separated trauma kit

now Bio or Mechanical

Turtlelord26 committed:
Separated survival packages

Turtlelord26 committed:
Cutting survival packages

 as we don't have those features yet anyway

Turtlelord26 committed:
Rekey kit prose

rosebrallier committed:
Merge pull request #983 from bleehu/MH/tools/clean-firearms

added prose for firearm tool.

Turtlelord26 committed:
Added prose

Electronic locks kit

Turtlelord26 committed:
Merge branch 'rosebrallier-backgrounds' into Turtlelord26-Tool_Sets

Turtlelord26 committed:
Absorbing the plasma cutter from items doc

Turtlelord26 committed:
Remove items absorbed into tool set document

Removed bolt cutters and plasma cutter.

Turtlelord26 committed:
Musical Instrument Care Supplies renamed as so

No longer reads as supplying an instrument too, for clarity. We could just as easily go the other way.

Turtlelord26 committed:
Added rosin to instrument supplies

Turtlelord26 committed:
Audial -> Auditory

Audial sounded nice though
